modules/shortcodes/instagram.php

```diff
@@ -191,9 +192,13 @@ function jetpack_instagram_handler( $matches, $atts, $url ) {
 			true
 		);
 		// there's a script in the response, which we strip on purpose since it's added by this ^ script
+		/*
 		$ig_embed = preg_replace( '@<(script)[^>]*?>.*?</\\1>@si', '', $response_body->html );
-
 		return $ig_embed;
+		*/
```

Comment:
Could these lines be removed instead of commented out?

Reply:
Yap, can be removed.

I just left it there for reference